authority/voting/server/state.go

```diff
@@ -647,7 +668,7 @@ func (s *state) sendVoteToPeer(peer *config.AuthorityPeer, vote []byte, epoch ui
 // for our link layer wire protocol as specified by
 // the PeerAuthenticator interface in core/wire/session.go
 func (s *state) IsPeerValid(creds *wire.PeerCredentials) bool {
-	var ad [eddsa.PublicKeySize]byte
+	var ad [publicKeyHashSize]byte
 	copy(ad[:], creds.AdditionalData)
```

Comment:
does this crash if there's more bytes in `creds.AdditionalData` than `publicKeyHashSize` ?